### DISCUSSION TITLE
Color blindness

### DISCUSSION THREAD

**Discussion by Lobaevalex** [apache/airflow#32813] [Ideas]
Hey!
Creating an awesome product from functionality perspective should not be the only goal for almost a decade. I mean UI. It's not really convenient, but now I want to pay attention on a real problem - it's almost impossible to use for people with color blindness or other color deficiency.
I'm working as a Data Engineer for 5 years and currently every day I'm working with Airflow and it's a nightmare, absolute nightmare to use it for me because I have some kind of "Deuteranopia" when I have issues in recognising between red and green. Therefore I have real issues with identifying if the job ran successfully or failed. I even have glasses with special lenses which helps me, but with the help of it then I can't recognise between running and failed job. In other words no one ever thought about this issue, because there are plenty of color pallets which would be great for everyone and the ones which are used in Airflow now I would name horrible. 
I think it's time to think about it. There is number of pallets could be found on the internet to understand which colours to use. Tableau could help here: https://www.tableau.com/blog/examining-data-viz-rules-dont-use-red-green-together
But would even better to think about using different elements for different statuses. For example:
- Success - green circle
- Failed - red square
- Running - yellow triangle
- Skip - empty white circle
- ...
If you are ready to continue the conversation on this topic I could prepare more realistic cases with real examples

**Comment by andresitorusz** [comment]
Sorry, didn't mean to offend. Did customizing the status color in Airflow not help?

Ref: https://airflow.apache.org/docs/apache-airflow/stable/howto/customize-ui.html#customizing-state-colours
